packages/electrode-archetype-react-app/config/archetype.js

```diff
@@ -121,20 +137,20 @@ function loadDev() {
     devDir,
     devPkg,
     devRequire,
-    webpack: {
+    webpack: Object.assign({}, archetypeOptions.webpack, {
       devHostname: "localhost",
       devPort: getInt(process.env.WEBPACK_DEV_PORT, 2992),
       testPort: getInt(process.env.WEBPACK_TEST_PORT, 3001),
       modulesDirectories: []
-    },
-    config: {
+    }),
```

Comment:
Do we want `devHostname` and the other options here to be configurable? Because the current order would cause these keys to always be overwritten. I think we want:

```js
Object.assign({
  devHostname: "localhost",
 ...
}, archetypeOptions.webpack)
```